docs/repositories.md

```diff
@@ -171,7 +181,18 @@ If you wish to avoid this, you may explicitly specify which source to search in
 package.
```

Comment:
`source =` should restrict this to only that repository -- if we're doing otherwise, that's a bug.

Reply:
It's difficult for me to assert this, because my private `pypi` is set to redirect to `pypi.org` on missing packages.

Since this closely touches security, there really should be a unit test that asserts it.